src/api.js

```diff
@@ -12,4 +12,19 @@ export const getArticles = (page) => {
 export const getUsers = () => {
     return newsApi.get('/users')
     .then(res => res.data.users);
+}
+
+export const getArticleById = (id) => {
+    return newsApi.get(`/articles/${id}`)
+    .then(res => res.data.article);
+}
+
+export const patchArticleById = (id, data) => {
+    return newsApi.patch(`/articles/${id}`, data)
+    .then(res => res.data.article);
+}
+
+export const getArticleCommentsById = (id) => {
+    return newsApi.get(`/articles/${id}/comments`)
+    .then(res => res.data.comments);
```

Comment:
Please only include relevant code in a pull request.

Reply:
My bad, I thought this was all the functionality for the article page did not realise there were other tickets for adding certain things.